pocket_lab/pl_gui_views.c

```diff
@@ -394,13 +394,15 @@ void pl_gui_perform_dmm_read(void)
 		read_cntr = 0;
 		for (cnt = 0; cnt < pl_gui_dmm_chn_cnt; cnt++) {
 			ibuf[0] = '\0';
-			if (lv_obj_get_state(pl_gui_dmm_chn_checkbox[cnt]) == LV_STATE_CHECKED) {
+			if (lv_obj_get_state(pl_gui_dmm_chn_checkbox[cnt]) == (LV_STATE_CHECKED |
```

Comment:
What is the reason behind this change? Can we have a short description in the commit message?